ui/packages/shared/profile/src/ProfileIcicleGraph/index.tsx

```diff
@@ -94,18 +141,29 @@ const ProfileIcicleGraph = ({
       return;
     }
     setActionButtons(
-      <>
-        <Button
-          color="neutral"
-          onClick={() => setNewCurPath([])}
-          disabled={curPath.length === 0}
-          variant="neutral"
-        >
-          Reset View
-        </Button>
-      </>
+      <div className="flex w-full justify-end gap-2 pb-2">
+        <div className="flex w-full items-center justify-between space-x-2">
+          {table !== undefined && (
+            <>
+              <GroupByDropdown groupBy={groupBy} toggleGroupBy={toggleGroupBy} />
+              <SortBySelect compareMode={compareMode} sortBy={localSortBy} setSortBy={setSortBy} />
+            </>
+          )}
+          <div>
+            <label>&nbsp;</label>
```

Comment:
This can be removed by adding a `gap-2` class to the parent element.

Reply:
Seems like `gap-2` on its own isn't helpful but in combination with `inline-block` it pushes the `Reset View` button down. In fact, `inline-block` is all that's needed.